src/components/custom-tom-select.scss

```diff
@@ -135,7 +135,7 @@
   vertical-align: middle;
   display: inline-block;
   padding: 0 6px;
-  border-left: 1px solid var(--rms-colors-neutral-80);
+  border-left: 1px solid var(--rm-color-neutral-80);
   border-radius: 0 2px 2px 0;
   box-sizing: border-box;
   margin-left: 6px;
```

Comment:
@Jeremy-Walton Can we use our tokens here instead of the static styles for margin.